test/unit/init/App.test.ts

```diff
@@ -22,4 +25,9 @@ describe('An App', (): void => {
     await expect(app.stop()).resolves.toBeUndefined();
     expect(finalizer.finalize).toHaveBeenCalledTimes(1);
   });
+
+  it('can check its clusterManager for the threading mode.', async(): Promise<void> => {
+    await expect(app.start()).resolves.toBeUndefined();
+    expect(app.clusterManager).toBe('CM');
```

Comment:
```suggestion
    expect(app.clusterManager).toBe(clusterManager);
```
`toBe` checks exact equality so can be done like this.